daemon/execdriver/windows/run.go

```diff
@@ -68,6 +66,7 @@ func (d *Driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 	cu := &containerInit{
 		SystemType:              "Container",
 		Name:                    c.ID,
+		Owner:                   "docker",
```

Comment:
Owner here is a tag used by the HCS to distinguish what management platform created the container. It allows us to differentiate between different management platforms. (Remember that there is native in-box management as well.)